EDI/Operation/04_ManageLineItemsFileStt.md

```diff
@@ -0,0 +1,185 @@
+
+
+## Prepare data mapping for the line items
+
+Prepare the data mapping for the line items so that the relevant data for the mapping is available to retrieve the line items.
+
+#### Prerequisites
+
+- You have defined both the definitions for the header and the line items, see [Create basic definition settings](./01_ManageDefinitions.md#create-basic-definition-settings).
+- You have prepared the line item export, see [Prepare line item export](./02_ManageHeaderFileSett.md#prepare-line-item-export).
+
+
+#### Procedure
+
+*DataHub Exporter > Exports*
+
+![Export definitions](../../Assets/Screenshots/EDI/Operation/ExportDefinitions.png "[Export definitions]")
+
+1. Click the header export definition for which you want to prepare the data mapping for the line items. 
+    The *Edit definition "Definition name"* view is displayed. The *Settings* tab is displayed by default.
+
+2. Click the *Data sources* tab.   
+    The *Data sources* tab is displayed.  
+
+    ![Data sources tab](../../Assets/Screenshots/EDI/Operation/DataSourcesListWithEntries.png "[Data sources tab]")
+
+3. Click the data source, for which you want to map the segment for the line items.   
+    - The *Map attributes* wizard window is displayed.     
+    - In the *Destination attribute* column, a list of all EDI segment rows is displayed for which you have defined a string attribute.    
+    - The *EDI export SG26/SG27* entry is available.
+
+        ![SG26 Mapping](../../Assets/Screenshots/EDI/Operation/ExportDefMapSG26Segment.png "[SG26 Mapping]")
+
+6. Click the *EDI Export SG26/SG27* destination attribute and define the mapping.    
+   The settings section is displayed on the right.
+
+    ![SG26 Mapping settings](../../Assets/Screenshots/EDI/Operation/ExportDefSG26MappingKonfig.png "[SG26 mapping settings]")
+   
+     -  Click the *Extension* drop-down list and select the **XML SubExport extension**.   
+     Both the "Source attribute" and the *Configuration* section is displayed in the left part of the *Settings* section.
+
+     - Click the *Override change tracking mode* drop-down list and select the desired one.
+
+     -  On the left part of the *Settings* section, click the *Source attribute* drop-down list and select **Positions** or another attribute that represents a line item. 
+
+     -  In the *Configuration* section, click the *Subexport definition* drop-down list and select the required line item definition, for example **EDIFACT INVOIC export items customer 12.**
+     
+        ![SG26 Mapping settings](../../Assets/Screenshots/EDI/Operation/ExportDefSG26MappingDone.png "[SG26 mapping settings]")  
+
+7. Click the [SAVE] button in the top right corner of the wizard.   
+    The *Please select a destination attribute in the grid on the left to edit the mapping* text is displayed.
```

Comment:
Evtl. message verallgemeinen? Ich vermute, es ist eine "Einladung", weitere Mappings zu konfigurieren. So was wie "A message to configure further mappings is displayed", o.ä.?

Reply:
Der text steht auf dem screen.